### ISSUE TITLE
Convert dot-notation to proper nesting

### ISSUE THREAD

**Issue by StevenLangbroek** [danjesus/gulp-phrase#4]
Right now the output is for example:

```json
{
    "faq.group0.entry0.answer": "some content"
}
```

ideally, this would be:

```json
{
    "faq0": {
        "group0": {
            "entry0": {
                "answer": "some content"
            }
        }
    }
}
```

**Comment by danjesus** [comment]
@StevenLangbroek  how are you doing this today? You download directly from phraseapp so?

**Comment by StevenLangbroek** [comment]
Right now the entire project still lives in Ruby on Rails, so we download YAML files, and RoR takes care of the nesting (among other things).

**Comment by danjesus** [comment]
Got it, for now I'm keeping the simple design just downloading simple_json.

I will make improvements to the project yet.

But if you use rails can use the phrase gem to automate the process.

Have you taken a look at this link? http://docs.phraseapp.com/guides/setup/localize-ruby-on-rails-applications-with-phraseapp/

Best

**Comment by StevenLangbroek** [comment]
Yeah that's what we're using for now, but we want to rip the app out of
rails and just serve it from nginx. We're still missing some bits of
functionality thought (smart i18n fallbacks for example), and probably
sticking to it for now.

Op zo 8 mrt. 2015 om 00:07 schreef Dan Jesus <notifications@github.com>:

**Comment by danjesus** [comment]
I understand now, I'm working on some changes soon we will have more resources to support this lib.
